lib/howl/buffer.moon

```diff
@@ -295,9 +305,15 @@ class Buffer extends PropertyObject
     @viewers -= 1
 
   _associate_with_file: (file) =>
+    scope = @_config_scope
     @_file = file
+    config.copy scope, @_config_scope
```

Comment:
Yes that's what would happen - I'm not saying it's right or wrong, just that it's closer to the preexisting behavior because the proxy object is attached to the buffer object, afaict.

In terms of what's right, if you did set some config for the buffer before `save-as`, you'd want that to be preserved after the `save-as` operation, so a clobber is reasonable IMO. Some kind of merge is also reasonable, but not clearly defined yet.